mysql-test/suite/innodb/r/innodb_stats_method,NULLS_UNEQUAL.rdiff

```diff
@@ -0,0 +1,32 @@
+--- innodb_stats_method.result
++++ innodb_stats_method,NULLS_UNEQUAL.result
+@@ -8,15 +8,15 @@
+ mysql.innodb_index_stats WHERE database_name= 'test'
+ AND table_name = 't1';
+ index_name	stat_value	stat_description
+-GEN_CLUST_INDEX	64	DB_ROW_ID
++GEN_CLUST_INDEX	64	DB_ROW_ID NULLS_UNEQUAL
+ GEN_CLUST_INDEX	1	Number of leaf pages in the index
+ GEN_CLUST_INDEX	1	Number of pages in the index
+-f1	64	f1
+-f1	64	f1,DB_ROW_ID
++f1	64	f1 NULLS_UNEQUAL
++f1	64	f1,DB_ROW_ID NULLS_UNEQUAL
+ f1	1	Number of leaf pages in the index
+ f1	1	Number of pages in the index
+-f2	33	f2
+-f2	64	f2,DB_ROW_ID
++f2	64	f2 NULLS_UNEQUAL
++f2	64	f2,DB_ROW_ID NULLS_UNEQUAL
```

Comment:
These results are the same as with `NULLS_IGNORED`. Could you come up with some data where all 3 variants would return different results?